src/components/common/Footer.tsx

```diff
@@ -0,0 +1,96 @@
+// Modified from: https://github.com/contentlayerdev/website/blob/main/src/components/common/Footer.tsx
+import { FC } from 'react';
+import Link from 'next/link';
+
+import { Logo } from './Logo';
+import { Icon } from './Icon';
+import { isExternalUrl } from '../../utils/helpers';
+import config from '../../config';
+import * as routes from '../../routes';
+
+const content = {
+  note: (
+    <p className="mt-2">
+      Made with 💜 by{' '}
+      <Link href={config.raydiantUrl}>
+        <a
+          className="hover:text-slate-700 dark:hover:text-slate-300"
+          target="_blank"
+          rel="noreferrer"
+        >
+          Raydiant
+        </a>
+      </Link>
+    </p>
```

Comment:
Torn with saying this part. I like it for OS projects but might be a little too "cute" for us

Reply:
Yeah, fair call. Removed here: https://github.com/mirainc/developer-docs/pull/7/commits/5093b1675b6b3df39ee02455395141c40e928588